react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+const pokemon = pokedex.map(pokedex =>
+  <li key ={pokedex.number}>{pokedex.name}</li>
+);
```

Comment:
This works but you should use a different name for the parameter, it is very confusing to use the same name for the parameter as the original array.
```JSX
const pokemon = pokedex.map(pokemon =>
  <li key ={pokemon.number}>{pokemon.name}</li>
);
```

I'd also recomend changing the name of the const as well, again it will still work with the same names but it can be confusing. Something like `const pokemonElements` or something would be better.